Documentation/WhenToUseBabylonNative.md

```diff
@@ -0,0 +1,194 @@
+-   [I prefer more native-like technologies.](#do-you-have-exceptional-rendering-requirements-such-as-aaa-quality-graphics-or-extreme-performance-needs)
+-   [I prefer more Web-like technologies.](#do-you-already-have-existing-expertise-andor-preferences-for-react-native-or-do-you-need-native-capabilities-that-arent-provided-by-normal-web-apis-such-as-ar-on-ios)
+
+#### Do you have exceptional rendering requirements such as AAA-quality graphics or extreme performance needs?
+-   [I want things to look good, but I don't need extreme graphics or performance.](#babylon-native-is-probably-a-good-choice-for-your-project)
+-   [I need AAA-quality graphics and/or exceptional performance.](#there-may-be-other-choices-better-suited-to-your-project-than-babylon-native-technologies)
+
+#### Do you already have existing expertise and/or preferences for React Native, or do you need native capabilities that aren't provided by normal Web APIs (such as AR on iOS)?
+-   [Yes.](#babylon-react-native-is-probably-a-good-choice-for-your-project)
+-   [No.](#there-may-be-other-choices-better-suited-to-your-project-than-babylon-native-technologies)
+
+#### Is your app built on React Native, another JavaScript-using platform (including Web or hybrid app platforms), or non-JavaScript technologies?
+-   [My app is a React Native app.](#babylon-react-native-is-probably-a-good-choice-for-your-project)
+-   [My app's platform uses JavaScript, but it isn't made with React Native.](#there-may-be-other-choices-better-suited-to-your-project-than-babylon-native-technologies)
+-   [My app's platform doesn't inherently use JavaScript.](#do-you-have-exceptional-rendering-requirements-such-as-aaa-quality-graphics-or-extreme-performance-needs)
+
+***
+
+## Babylon Native is probably a good choice for your project!
+
+Congratulations! If you ended up here, then we think Babylon Native may
+be a suitable choice for adding 3D to your project. Typically, 
+applications for which Babylon Native is well-suited tend to be developed
+as true native apps (C++, C#, etc.) which require 3D rendering as a 
+feature, though not necessarily the focus. Examples might include the
+following:
+
+-   A presentation app written in C# that needs to be able to render 3D 
+    as well as 2D content.
+-   A scientific simulation utility written in Java that wants to be able 
+    to render data in 3D.
+-   A 3D content generation tool written in C++ that wants to be able to 
+    preview its output.
+
+The main reason Babylon Native is well-suited for these scenarios is 
+because Babylon Native makes it possible to bring Babylon.js's 
+rendering capabilities into native applications with *extremely* minimal
+requirements regarding app and development infrastructure. In other words,
+Babylon Native is perhaps the simplest way to add Web-style 3D rendering
+to a native application without having to pivot that application to 
+*become* a Web app.
```

Comment:
nit: just to avoid "js`s"

```suggestion
The main reason Babylon Native is well-suited for these scenarios is 
because Babylon Native makes it possible to bring the rendering
capabilities of Babylon.js into native applications with *extremely* minimal
requirements regarding app and development infrastructure. In other words,
Babylon Native is perhaps the simplest way to add Web-style 3D rendering
to a native application without having to pivot that application to 
*become* a Web app.
```